C++/Navio/Common/gpio.cpp

```diff
@@ -137,37 +132,42 @@ void Pin::toggle()
     write(!read());
 }
 
-int Pin::getRaspberryPiVersion() const
+int Pin::getPeriBase() const
+{
+    size_t rpi_version = getRaspberryPiVersion();
+
+    switch (rpi_version) {
+      case 1:
+        return BCM2708_PERI_BASE;
+      case 2:
+        return BCM2709_PERI_BASE;
+      case 3:
+        return BCM2835_PERI_BASE;
+      default:
+        warnx("Unexpected version. Set PERI_BASE to default as for RPI2/3\n");
```

Comment:
`Set PERI_BASE to default as for RPI2/3` I was confused about this, please write such that it's obvious it's you set PERI_BASE to default. Not like you are asking someone to do this. `PERI_BASE` should not be in a warning.